src/development/stack.yml

```diff
@@ -333,6 +336,8 @@ services:
     - --providers.file.filename=/dynamic.yml #DARGSTACK-REMOVE
     - --providers.file.watch=true #DARGSTACK-REMOVE
     - --log.level=DEBUG #DARGSTACK-REMOVE
+    - --experimental.localPlugins.body-forward-auth.moduleName=github.com/PatrickMi/body-forward-auth
+    # - --experimental.plugins.body-forward-auth.version=0.1.1
```

Comment:
Seems like Go is using an own versioning scheme with a prefixed `v` after all (https://go.dev/doc/modules/version-numbers). Maybe the plugin download will then work?
I recommended not using `v` as a prefix, because the automated releases for maevsi do not use it. Forgot about Go's custom scheme.

Reply:
Thanks for that! It works now with the GitHub repository :-)